setup.py

```diff
@@ -26,6 +26,7 @@
     "setproctitle==1.2.2",  # Gives the chia processes readable names
     "sortedcontainers==2.4.0",  # For maintaining sorted mempools
     "websockets==8.1.0",  # For use in wallet RPC and electron UI
+    # TODO: when moving to click 8 remove the pinning of black noted below
```

Comment:
Just to note, this won't actually be added as it is already present in `atari`.  This showing is just an artifact of the branch not having been caught up, which is fine.  Same below.